package.json

```diff
@@ -23,12 +29,16 @@
 		"@types/react": "^16.0.19",
 		"@types/react-native": "^0.49.5",
 		"@types/react-test-renderer": "^16.0.0",
+		"babel-core": "^6.26.0",
 		"babel-jest": "21.2.0",
 		"babel-preset-react-native": "4.0.0",
 		"concurrently": "^3.5.0",
 		"jest": "21.2.1",
 		"react-test-renderer": "16.0.0-beta.5",
 		"ts-jest": "^21.1.4",
+		"tslint": "5.8.0",
```

Comment:
i think we should decide if we're going to be pinning versions or getting the latest so that these are consistent

Reply:
I agree we should be consistent